p4-16/spec/P4-16-spec.mdk

```diff
@@ -5545,7 +5545,7 @@ match_kind {
 ~ End P4Example
 
 These identifiers correspond to the P4~14~ match kinds with the same
-names. The semantics of these annotations is actually not needed to
+names. The semantics of these match kinds is actually not needed to
```

Comment:
This is not directly related to adding `@match` as another predefined annotation, but I noticed this potential misuse of the word "annotations" here while looking for all occurrences of the word "annotations" and similar words, and thought I would try improving on it.